spec/V4/forum_scraper_spec.rb

```diff
@@ -1,5 +1,4 @@
 require_relative '../spec_helper'
-require_relative '../../lib/V4/forum_scraper'
```

Comment:
I removed all these require_relatives from the specs since the spec_helper is already loading in the main module, which includes each scraper.